rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -222,6 +222,19 @@ usb_handler_thread::usb_handler_thread()
 			found_skylander = true;
 		}
 
+		if (desc.idVendor == 0x1209 && desc.idProduct == 0x2882)
```

Comment:
Why are you just dropping this between two entries in the "Portals" section?
This should have its own section

Reply:
Since this emulates music devices, ill throw it in the music devices section